src/plugins/graph/models/graph-model.ts

```diff
@@ -73,7 +73,9 @@ export const GraphModel = TileContentModel
     plotBackgroundLockInfo: types.maybe(types.frozen<BackgroundLockInfo>()),
     // numberToggleModel: types.optional(types.union(NumberToggleModel, null))
     showParentToggles: false,
-    showMeasuresForSelection: false
+    showMeasuresForSelection: false,
+    xAttributeLabel: types.optional(types.string, kDefaultAxisLabel),
+    yAttributeLabel: types.optional(types.string, kDefaultAxisLabel)
```

Comment:
We have a `defaultAxisLabels` setting already in the unit configuration. I think these should be initialized from that setting so the default labeling can be adjusted the the different projects (the new default can be set in app-config.json).  Previously `defaultAxisLabels` was used only for _empty_ graphs, but it seems reasonable that it should be the default here too.

Reply:
I did look into `defaultAxisLabels` a bit, but just went with "axis label" because the story has this pretty blunt second bullet point: "Default text reads axis label". I'm not opposed to using `defaultAxisLabels`, though I'm not immediately sure how to access the unit settings from a model. Do you know of any models that use unit settings as their defaults?

Reply:
Hm, I hadn't thought about that being a problem, but you're right the usual useAppConfig is not going to work there.  I guess we can leave it non-configurable until told otherwise.

Reply:
I was able to figure out a partial solution to this. `createGraphModel` is how new graphs are created, and it has access to `appConfig`, so it can get `defaultAxisLabels`. This is less robust than making `defaultAxisLabels` the default of the model, but it should work for new graphs. It won't work for graphs in existing documents, which will get "axis label" no matter which unit they are in.